pkg/queue/request_metric.go

```diff
@@ -62,19 +62,21 @@ var (
 )
 
 type requestMetricsHandler struct {
-	next     http.Handler
-	statsCtx context.Context
+	next                  http.Handler
+	statsCtx              context.Context
+	reportLatencyInHeader bool
 }
 
 type appRequestMetricsHandler struct {
-	next     http.Handler
-	statsCtx context.Context
-	breaker  *Breaker
+	next                  http.Handler
+	statsCtx              context.Context
+	breaker               *Breaker
+	reportLatencyInHeader bool
 }
 
 // NewRequestMetricsHandler creates an http.Handler that emits request metrics.
 func NewRequestMetricsHandler(next http.Handler,
-	ns, service, config, rev, pod string) (http.Handler, error) {
+	ns, service, config, rev, pod string, reportLatencyInHeader bool) (http.Handler, error) {
```

Comment:
you need to change this func call in the test as well since the signature changed: `pkg/queue/request_metric_test.go`